### ISSUE TITLE
Display garbled problem

### ISSUE THREAD

**Issue by wongFanx** [zolrath/obsidian-auto-link-title#2]
Identify links: https://zhidao.baidu.com/question/509882629.html

Display effect:
![image](https://user-images.githubusercontent.com/14831158/115510189-01064a80-a2b2-11eb-87f6-4ce9607924be.png)

**Comment by zolrath** [comment]
Ah, thank you very much! I'll look into the encoding of the fetched titles.

**Comment by zolrath** [comment]
Unfortunately the public CORS proxies that are available don't handle Chinese characters correctly.
I'll look into providing the option to customize the proxy and allow you to run one yourself.

**Comment by zolrath** [comment]
The local scraping solution I mentioned in the topic also fixes this. Need to test mobile.

```
[图书馆学、文献学、情报学、档案学、目录学之间的区别，答案力求精辟、专业_百度知道](https://zhidao.baidu.com/question/509882629.html)
```

**Comment by zolrath** [comment]
Fixed for Desktop on 1.2.0
Mobile still relies on the CORS proxy that doesn't support these characters.
